From 8614051a1ff9856c3de932045dc149e758c2b0b1 Mon Sep 17 00:00:00 2001
From: Adrian Ratiu <adrian.ratiu@collabora.com>
Date: Fri, 12 Apr 2024 17:08:09 +0300
Subject: [PATCH] CMakeLists.txt: fix pkgconfig install path

pkgconfig pc files need to go under libdir instead of datadir.

Suggested-by: Sam James <sam@gentoo.org>
Upstream-Status: Backport [upstream commit 8614051]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9d533ab..aecf4e5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,4 +59,4 @@ configure_file(
 )
 
 install(FILES ${CMAKE_BINARY_DIR}/libtypec.pc
-    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/pkgconfig)
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
-- 
2.43.2

